bundles/core/src/test/java/com/adobe/cq/wcm/core/components/Utils.java

```diff
@@ -92,4 +98,33 @@ public static String getTestExporterJSONPath(String testBase, String testResourc
         return testBase + "/exporter-" + FilenameUtils.getName(testResourcePath) + ".json";
     }
 
+    /**
+     * Sets the data layer context aware configuration of the AEM test context to enabled/disabled
+     *
+     * @param context The AEM test context
+     * @param enabled {@code true} to enable the data layer, {@code false} to disable it
+     */
+    public static void enableDataLayer(AemContext context, boolean enabled) {
```

Comment:
I would put this in `com.adobe.cq.wcm.core.components.testing.Utils`. We should actually get rid of this class altogether and move everything to `core.wcm.components.junit.core` bundle.

Reply:
I would leave it like that for this PR and would refactor/move this class in a separate PR: I created an issue (https://github.com/adobe/aem-core-wcm-components/issues/994) to track it.